draft-ietf-quic-transport.md

```diff
@@ -678,6 +691,22 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+A sender can combine multiple QUIC packets (typically a Cryptographic Handshake
+packet and a Protected packet) into one UDP packet, thereby reducing the number
+of UDP packets required to be emitted when application data can be sent during
+the handshake.
```

Comment:
Add: "A packet with a short header doesn't include a length, so it has to be the last packet included in a UDP datagram."

Reply:
Done in 9dd629e.